modules/installation-configuration-parameters.adoc

```diff
@@ -1094,6 +1114,36 @@ This example defines no affinity groups:
       affinityGroupNames: []
 ----
 |String
+|`<machine-pool>.platform.ovirt.AutoPinningPolicy`
+| Optional. AutoPinningPolicy defines the policy to automatically set the CPU and NUMA settings including pinning to the host for the instance. When the field is omitted, the default is `none`. Supported values: `none`, `resize_and_pin`.
+
+[source,yaml]
+----
+<machine-pool>:
+autoPinningPolicy:
+description: AutoPinningPolicy defines the policy to automatically set the CPU and NUMA including pinning to the host for the instance. When the field is omitted the default will be "none".
+  enum:
+    - none
+    - resize_and_pin
+  type: string
+----
+
+|String
+|`<machine-pool>.platform.ovirt.hugepages`
+|Optional. Hugepages is the size in KiB for defining hugepages in a VM. Supported values: 2048 or 1048576.
```

Comment:
Maybe give a link to the RHV docs about hugepages? not a must, only if you think this is needed